packages/core/src/App/Containers/Layout/header/dashboard-platform-header.jsx

```diff
@@ -0,0 +1,252 @@
+import { connect } from 'Stores/connect';
+import { BinaryLink } from '../../../Components/Routes';
+
+const Divider = () => {
+    return <div className='dashboard-platform-header__divider' />;
+};
+
+const MyApps = () => {
+    const history = useHistory();
+
+    return (
+        <div className='dashboard-platform-header__my-apps' onClick={() => history.push(routes.dashboard)}>
+            <Icon icon='IcWalletMyApps' size={16} />
+            <Text size='s' line_height='xxl' weight='bold' className='dashboard-platform-header__my-apps--title'>
+                {localize('My apps')}
+            </Text>
+            <Divider />
+        </div>
+    );
+};
+
+const PlatformInformation = ({ app_routing_history }) => {
+    return (
+        <div className='dashboard-platform-header__platform-information'>
+            <Icon
+                className='dashboard-platform-header__platform-information--icon'
+                icon={getPlatformInformation(app_routing_history).icon}
+                size={24}
+            />
+            <Text size='s' weight='bold' line_height='xxl'>
+                {getPlatformInformation(app_routing_history).header}
+            </Text>
+        </div>
+    );
+};
+
+const ShowReports = () => {
+    return (
+        <div className='dashboard-platform-header__report'>
+            <BinaryLink id='db_reports_tab' key='db_reports_tab' to={routes.reports}>
+                <Icon icon='IcReports' size={20} />
+            </BinaryLink>
+        </div>
+    );
+};
+
+const ShowNotifications = ({ is_notifications_visible, notifications_count, toggleNotifications }) => {
+    return (
+        <div className='dashboard-platform-header__notification'>
+            <ToggleNotifications
+                count={notifications_count}
+                is_visible={is_notifications_visible}
+                toggleDialog={toggleNotifications}
+                tooltip_message={<Localize i18n_default_text='View notifications' />}
+            />
+        </div>
+    );
+};
+
+const ShowSettings = ({ toggleSettings }) => {
+    return (
+        <div className='dashboard-platform-header__setting'>
+            <Popover alignment='top'>
+                <a
+                    id='dt_settings_toggle'
+                    onClick={toggleSettings}
+                    className='dashboard-platform-header__setting--icon'
+                >
+                    <Icon icon='IcGearLight' size={22} />
+                </a>
+            </Popover>
+        </div>
+    );
+};
+
+const AccountBalance = ({ balance, currency }) => {
+    return (
+        <div className='dashboard-platform-header__balance'>
+            <Divider />
+            <Icon icon='IcWalletDerivApps' size={28} className='dashboard-platform-header__balance--icon' />
+            <Text size='s' line_height='xxl' weight='bold' color='profit-success'>
+                {(typeof balance !== 'undefined' || !currency) && `${balance} ${getCurrencyDisplayCode(currency)}`}
```

Comment:
please extract this condition to a variable

Reply:
@mustofa-binary I replace it with the `Money` component.